mentatsync/storage/sql/dbconnect.py

```diff
@@ -91,7 +91,8 @@ def UUID():
     metadata,
     Column("userid", UUID(), primary_key=True, nullable=False),
     Column("trnid", UUID(), primary_key=True, nullable=False),
-    Column("idx", Integer, primary_key=True, nullable=False),
+    Column("idx", Integer, primary_key=True, nullable=False,
+           autoincrement=False),
```

Comment:
@grigoryk if you're curious, this was the cause of the storage weirdness I mentioned - MySQL assumed that since I have an integer as part of my primary key, of course I must want it to auto-increment :-/